span-normalizer/span-normalizer-api/src/main/avro/SpanMetadata.avdl

```diff
@@ -0,0 +1,6 @@
+@namespace("org.hypertrace.core.spannormalizer")
+protocol SpanMetadataProtocol {
```

Comment:
move it to raw-spans-grouper

Reply:
There is plan to merge grouper and normaliser, so keeping it together for now.